.eslintrc

```diff
@@ -51,7 +50,7 @@
     "react/no-unused-prop-types": "off",
     "class-methods-use-this": "error",
     "import/no-extraneous-dependencies": [
-      "error",
+      "off",
```

Comment:
Added this to prevent errors when importing `vitest`. I believe we can configure `vitest` to work globally like `jest`, but didn't spend too much time trying to figure out how haha

Reply:
How come eslint still errors on this when `vitest` is indeed declared as a devDependency?

(I'm not a vitest nor eslint configuration expert, it's a genuine question)

Reply:
Hey Pavel! I believe it's a rule to prevent the import of dev dependencies that causes this error. I think this rule exists because dev dependencies are typically stripped in production, but since we only use them in tests, it shouldn't be an issue (I think!)

Reply:
FYI config for global vitest is pretty easy
https://vitest.dev/config/#globals
```
// tsconfig.json
{
  "compilerOptions": {
    "types": ["vitest/globals"]
  }
}
```

```
// vite.config.ts
import { defineConfig } from 'vitest/config'

export default defineConfig({
  test: {
    globals: true,
  },
})
```

As for the lint rule, I don't really get why it doesn't work. `devDependencies: false` should never error when devDeps are imported

Reply:
Ah, it looks like we have `devDependencies: true` which might be why it's giving us an error then